package.json

```diff
@@ -53,7 +53,7 @@
     "react-native-collapsible": "^1.6.0",
     "react-native-gesture-handler": "~2.8.0",
     "react-native-maps": "1.3.2",
-    "react-native-reanimated": "^2.13.0",
+    "react-native-reanimated": "~2.12.0",
```

Comment:
Should we add `expo doctor` to our CI?

Reply:
it's an interesting idea, I like it!

Reply:
Tracked in https://github.com/stevekuznetsov/avalanche-forecast/issues/45